src/konnektor/network_planners/generators/clustered_network_generator.py

```diff
@@ -40,7 +40,7 @@ def __init__(
         clusterer: ComponentsDiversityClusterer = ComponentsDiversityClusterer(
             featurize=RDKitFingerprintTransformer(), cluster=KMeans(n_clusters=3)
         ),
-        mappers: Union[AtomMapper, list[AtomMapper]] = None,
+        mappers: Union[AtomMapper, list[AtomMapper]] = None,  # include None in this union?
```

Comment:
I confess, I really don't know what the correct type annotation is if there is a default None. :(

Reply:
I agree. I don't understand if there are cases where you would want to allow not passing through an AtomMapper. Also the score, line below, is optional, while not optional in the parent class. Lines 98 and 106 would be the cases where you don't need the AtomMapper and scorer, but I don't understand yet what those would do. I have to look into this in more detail.

Reply:
hm, it actually seems like `mappers` are _not_ required in the parent's parent class, `NetworkPlanner`. I opened an issue to resolve this, since this is no longer a docstrings edit: https://github.com/OpenFreeEnergy/konnektor/issues/93